Flow.Launcher/Languages/en.xaml

```diff
@@ -366,6 +366,28 @@
     <system:String x:Key="showBadges">Show Result Badges</system:String>
     <system:String x:Key="showBadgesToolTip">For supported plugins, badges are displayed to help distinguish them more easily.</system:String>
     <system:String x:Key="showBadgesGlobalOnly">Show Result Badges for Global Query Only</system:String>
+    <system:String x:Key="showBadgesGlobalOnlyToolTip">Show badges for global query results only</system:String>
+    <system:String x:Key="quickSwitchHotkey">Quick Switch</system:String>
+    <system:String x:Key="quickSwitchHotkeyToolTip">Enter shortcut to quickly navigate the path of a file dialog to the path of the current file manager.</system:String>
+    <system:String x:Key="quickSwitch">Quick Switch</system:String>
+    <system:String x:Key="quickSwitchToolTip">Quickly navigate to the path of the current file manager when a file dialog is opened.</system:String>
+    <system:String x:Key="autoQuickSwitch">Quick Switch Automatically</system:String>
+    <system:String x:Key="autoQuickSwitchToolTip">Automatically navigate to the path of the current file manager when a file dialog is opened. (Experimental)</system:String>
+    <system:String x:Key="showQuickSwitchWindow">Show Quick Switch Window</system:String>
+    <system:String x:Key="showQuickSwitchWindowToolTip">Show quick switch search window when file dialogs are opened to navigate its path (Only work for file open dialog).</system:String>
```

Comment:
I have been able to use it for file save dialogue as well, can you also confirm?